code/SCINGE.m

```diff
@@ -1,16 +1,29 @@
-function [ranked_edges, gene_influence] = SCINGE(Data,outdir,num_replicates,gene_list,param_list)
-
+function [ranked_edges, gene_influence] = SCINGE(gene_list,Data,outdir,num_replicates,param_list)
+% [ranked_edges, gene_influence] = SCINGE(gene_list,Data,outdir,num_replicates,param_list)
+% Standalone SCINGE implementation. 
+% Inputs:
+% gene_list = N x 1 cell array with list of relevant genes in the data set
+% Data = string representing the path of mat file containing the expression
+% data in the form of cell array X.
+% corresponding to above gene_list
```

Comment:
Does the line of the comments belong with the above line?  It looks incomplete possibly.

Reply:
Yes, I'll fix that.